docs/extend/index.md

```diff
@@ -1,24 +1,14 @@
 ---
+title: Docker Engine managed plugin system
 description: Develop and use a plugin with the managed plugin system
 keywords: "API, Usage, plugins, documentation, developer"
 ---
 
-<!-- This file is maintained within the docker/cli GitHub
-     repository at https://github.com/docker/cli/. Make all
-     pull requests against that repo. If you see this file in
-     another repository, consider it read-only there, as it will
-     periodically be overwritten by the definitive file. Pull
-     requests which include edits to this file in other repositories
-     will be rejected.
--->
```

Comment:
If we don't need these anymore; perhaps you can open a PR to remove them in all files.

(wondering though if this file vendored as-is, so this would still be visible?)

Reply:
yeah it's vendored as markdown, but they're in the _vendor directory and we have a contribution guideline outlining this. So I think it's fine to remove.